controllers/api.js

```diff
@@ -247,4 +247,12 @@ router
                 });
             }
         });
+    })
+
+    .use(function(err, req, res, next){
+        if (err) {
+            console.log(err);
+            res.status(500);
+            res[req.rt]({ error: err });
```

Comment:
What is `req.rt`? I have not seen that before and can not find it mentioned in the express documentation